src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline

```diff
@@ -106,14 +106,14 @@ function LibraryServiceClient(gaxGrpc, grpcClients, opts) {
   var port = opts.port || DEFAULT_SERVICE_PORT;
   var sslCreds = opts.sslCreds || null;
   var clientConfig = opts.clientConfig || {};
-  var appName = opts.appName || 'gax';
-  var appVersion = opts.appVersion || gax.version;
 
   var googleApiClient = [
-    appName + '/' + appVersion,
+    'gl-node/' + process.versions.node,
+    'gccl/' + require('../../package.json').version,
```

Comment:
gapic is given as `CODE_GEN_NAME_VERSION` constant below. 
fixed the `gccl` part. Thanks.